snippets/erlang.snippets

```diff
@@ -1,14 +1,6 @@
-# module and export all
+# module
 snippet mod
-	-module(${1:`vim_snippets#Filename('', 'my')`}).
-
-	-compile([export_all]).
-
-	start() ->
-	    ${0}
-
-	stop() ->
-	    ok.
+	-module(${1:`vim_snippets#Filename()`}).
```

Comment:
Could you make a new snippet for module that exports all? Possibly `modall` ?

Thanks